src/Command/InfectionCommand.php

```diff
@@ -171,7 +172,17 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $container = $this->getContainer();
-        $testFrameworkKey = $input->getOption('test-framework');
+
+        $bootstrap = $container->get('infection.config')->getBootstrap();
```

Comment:
`$container->get('infection.config')` is used twice, could you please extract to variable?